### ISSUE TITLE
Investigate dataset delete errors in production environment 3/24/16

### ISSUE THREAD

**Issue by bwitham** [ngageoint/hootenanny-ui#330]
Unfortunately this task is very vague with no clear plan how to tackle it, so define how you want to approach this before starting....  

Datasets are being deleted but the hoot db is left in an inconsistent state.  We need to find some way to prevent this from happening, regardless if its configuration related.  

Excerpt from a message from @qwyjibojones:

"It seems that durring a training session some Merged data sets were created. 
They later started cleaning up those datasets by deleting them in the UI. Some would go away, but some would remain with a zero size.  They can not remove the entry in the UI. For instance, when I try to remove the dataset associated with ID 2050, it goes red then re-appears in the UI.

The catalina out log shows these Errors:
ERROR:table "current_nodes_2050" does not exist
ERROR:table "current_ways_2050" does not exist
ERROR:table "current_relations_2050" does not exist
ERROR:table "changesets_2050" does not exist"

**Comment by bwitham** [comment]
duplicate of ngageoint/hootenanny#501
